ruby-sdk/lib/eppo_client/client.rb

```diff
@@ -43,6 +43,10 @@ def shutdown
       @core.shutdown
     end
 
+    def track(event_type, payload)
+      @core.track(event_type, payload)
+    end
```

Comment:
Is there a way to mark ruby functions as unstable? I would like to do that, so we can keep treating API as unstable and iterate on it without breaking changes

Reply:
@felipecsl I have resolved all of my comments except this one. What do you think about prefixing this function with `unstable_`?
```suggestion
    def unstable_track(event_type, payload)
      @core.track(event_type, payload)
    end
```

Reply:
hmmm I'm hesitant about it. i'm not aware of any built-in ruby features for marking as unstable. I'd say a comment about it should suffice, also as long as it's not yet in the public docs, then it in general shouldn't be used, but I think I'll defer to @leoromanovsky here

Reply:
renamed